aws/terraform/README.md

```diff
@@ -0,0 +1,112 @@
+## Pre-requisites
+
+### Terraform
+
+[Download](https://www.terraform.io/downloads.html) and install terraform 0.14.x.
+
+### AWS
+
+AWS should be [configured](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html) with a profile. The `AWS_PROFILE` environment should be set to the profile that has been configured.
+
+The [awscli](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) can be used as follows to 
+confirm that it has been configured properly by running:
+
+```
+aws sts get-caller-identity
+```
+
+which should output your account information.
+
+## Setup
+
+### Infrastructure stack
+
+The infra sub-project provides some pre-requisite infrastructure for the metaflow service. For more details see the [README](aws/terraform/infra/README.md)
+
+Copy `example.tfvars` to `prod.tfvars` (or whatever environment name you prefer) and update that `env` name and the `region` as needed.
```

Comment:
Minor nit (and we can address this in a later PR, noting here for posterity) - Maybe we can add some more context around `that env name and the region`.